src/Admin/AdminConsole/Models/OrganizationEditModel.cs

```diff
@@ -162,19 +162,18 @@ public OrganizationEditModel(Provider provider)
                 { "baseServiceAccount", p.SecretsManager.BaseServiceAccount }
             });
 
-    public Organization CreateOrganization(Provider provider, bool flexibleCollectionsSignupEnabled, bool flexibleCollectionsV1Enabled)
+    public Organization CreateOrganization(Provider provider, bool flexibleCollectionsV1Enabled)
     {
         BillingEmail = provider.BillingEmail;
 
         var newOrg = new Organization
         {
             // This feature flag indicates that new organizations should be automatically onboarded to
```

Comment:
Suggestion: give these comments an update to match the new actions (or lack thereof)

Reply:
I'm not sure exactly what you had in mind here but I updated the comments to be more specific, let me know what you think.

I also considered making the `Organization.FlexibleCollections` property read-only and always defaulting to `true`, but it's still used by the migration code which hasn't been deleted yet, so I maintained the current approach for now.